crates/typst/src/visualize/image/mod.rs

```diff
@@ -54,7 +54,7 @@ use crate::World;
 /// [gh-svg]: https://github.com/typst/typst/issues?q=is%3Aopen+is%3Aissue+label%3Asvg
 #[elem(scope, Show, LocalName, Figurable)]
 pub struct ImageElem {
-    /// Path to an image file.
+    /// Path to an image file, relative to the input file by default; if starting with `/`, it's relative to the `--root` path.
```

Comment:
For non-CLI users, they would not know what `--root` is. :confused:

Reply:
According to the [Usage](https://github.com/typst/typst/blob/main/README.md#usage) chapter of the readme, the tool is used only from CLI.

Reply:
It talks about the Typst web app in the same section.

Reply:
@laurmaedje true, I missed that. What is your proposal then, where to put this information? Which is quite important especially when you're writing a multi-file document.

Reply:
I think we could do something like this:
```rs
    /// Path to an image file.
    ///
    /// By default, the path is relative to the file where `image` is called.
    /// When starting with `/`, it becomes relative to the project root instead
    /// (configurable via `--root` in the CLI).
```
However, the above holds true for _all_ paths, be it image, import, json, etc. So perhaps it would be better to have this information somewhere else and link to it. Not sure where though, so I would be okay with having it here for now. We need for more guide-level docs in the future and then could reconsider this.

Reply:
@laurmaedje maybe then put it on the [Reference/Syntax](https://github.com/typst/typst/blob/main/docs/reference/syntax.md) page? Like make a new chapter "Specifying path" or just "Path".

But I don't know how the documentation build system works in this repo and how to produce links to static files from auto-generated.

Reply:
If there was a "Paths" section in the syntax page, it would be `[..]($syntax/#paths)`. Paths aren't really syntax, but maybe we could put it there.

Reply:
Please check d65f347defaa53280728252359df9bd5a0aa561b

* add "Paths" section
* provide reference to the "Paths" section in all modules which are trying to resolve a path

Reply:
I believe the discussion is resolved. Re-open if I'm wrong.